pyatlan/generator/templates/methods/attribute/a_p_i_path.jinja2

```diff
@@ -0,0 +1,27 @@
+
+        @classmethod
+        # @validate_arguments()
+        def create(cls, *, name: str, connection_qualified_name: str, apiPathRawURI: str, apiSpecQualifiedName: str) -> {{ entity_def.name }}.Attributes:
```

Comment:
I don't think we need all of these parameters:

- The `connection_qualified_name` can be calculated from the `api_spec_qualified_name`.
- The `name` can be calculated from the `api_path_raw_uri` (this must be unique within a spec anyway).

See the `creator(String, String)` method in the Java SDK's `com.atlan.model.assets.APIPath` for the logic 🙏

Reply:
I understand but according to the documentation isn't `If the name of your path does no give the URI of the endpoint it represents, be sure to also specify the apiPathRawURI.` means that name and api_path_raw_uri can have different value. How do i integrate something like this?

Reply:
Alright! Understood, Thanks! Fixed this in the code 👍